silero_vad_iterator.py

```diff
@@ -38,13 +38,18 @@ def __init__(self,
 
         self.model = model
         self.threshold = threshold
-        self.sampling_rate = sampling_rate
+        self.samplerate = samplerate
 
-        if sampling_rate not in [8000, 16000]:
+        if samplerate not in [8000, 16000]:
```

Comment:
what is the advantage of 8kHz Silero VAD Iterator? Is it faster or more accurate?